client/src/frontend/components/collecting/reading-group/edit/CollectionEditor/SortableCategories/parts/CategoryHeader.js

```diff
@@ -12,59 +12,53 @@ function CategoryHeader({
   onCategoryEdit,
   onCategoryRemove
 }) {
-  const [showDrawer, setShowDrawer] = useState(false);
-
-  function handleSuccess() {
-    setShowDrawer(false);
-    onCategoryEdit();
-  }
-
-  function handleCancel() {
-    setShowDrawer(false);
-  }
-
   function handleRemove() {
     onCategoryRemove(category);
   }
 
   return (
     <>
-      <header className="group-collection-editor__block group-collection-editor__block--category">
-        <h3 className="group-collection-editor__label group-collection-editor__label--category">
-          {category?.attributes.title}
-        </h3>
-        {dragProps && (
-          <div className="group-collection-editor__actions">
-            <CategoryRemove onRemove={handleRemove} />
-            <button
-              onClick={() => setShowDrawer(prevState => !prevState)}
-              aria-expanded={showDrawer}
-              className="group-collection-editor__action"
-            >
-              <IconComposer icon="annotate32" size="default" />
-              <span className="screen-reader-text">Edit category</span>
-            </button>
-            <div
-              {...dragProps.provided.dragHandleProps}
-              className="group-collection-editor__action"
-            >
-              <IconComposer icon="grabber32" size="default" />
+      <Collapse>
+        <header className="group-collection-editor__block group-collection-editor__block--category">
+          <h3 className="group-collection-editor__label group-collection-editor__label--category">
+            {category?.attributes.title}
+          </h3>
+          {dragProps && (
+            <div className="group-collection-editor__actions">
+              <CategoryRemove onRemove={handleRemove} />
+              <Collapse.Toggle className="group-collection-editor__action">
+                <IconComposer icon="annotate32" size="default" />
+                <span className="screen-reader-text">Edit category</span>
+              </Collapse.Toggle>
+              <div
+                {...dragProps.provided.dragHandleProps}
+                className="group-collection-editor__action"
+              >
+                <IconComposer icon="grabber32" size="default" />
+              </div>
             </div>
-          </div>
+          )}
+        </header>
+        {dragProps && (
+          <Collapse.Content>
```

Comment:
I like this idea, but I need to think through the math to get it to work. Somehow all good animations end up involving trig... As a more immediate fix, I added a prop `maxDuration` to `Collapse.Content`, so we can just pick a speed for anything that feels too slow.

There are two ways to speed things up globally-ish: One is to change the divisor on height (currently 700), but it's already pretty fast for very short things like one excerpt. The second is to adjust the default max value in the clamp. I set it at 1 sec because of some of the really long lists of excerpts, but I agree that it makes some of these medium height ones feel slow.

Reply:
The really long excerpts are pretty unusual, so I think we should focus on something that works best for medium-height content. I like the idea of reducing the default max clamp value. And we could keep the maxDuration prop as an override.

Reply:
I went with 500ms as the default max